replication_types.go

```diff
@@ -0,0 +1,37 @@
+package harbor
+
+import "time"
+
+// ReplicationExecution holds information about one replication execution.
+type ReplicationExecution struct {
+	ID         int64       `orm:"pk;auto;column(id)" json:"id"`
+	PolicyID   int64       `orm:"column(policy_id)" json:"policy_id"`
+	Status     string      `orm:"column(status)" json:"status"`
+	StatusText string      `orm:"column(status_text)" json:"status_text"`
+	Total      int         `orm:"column(total)" json:"total"`
+	Failed     int         `orm:"column(failed)" json:"failed"`
+	Succeed    int         `orm:"column(succeed)" json:"succeed"`
+	InProgress int         `orm:"column(in_progress)" json:"in_progress"`
+	Stopped    int         `orm:"column(stopped)" json:"stopped"`
+	Trigger    TriggerType `orm:"column(trigger)" json:"trigger"`
+	StartTime  time.Time   `orm:"column(start_time)" json:"start_time"`
+	EndTime    time.Time   `orm:"column(end_time)" json:"end_time"`
```

Comment:
What are the `orm` annotations for? Or is this copy&pasted from somewhere else (if so, please provide attribution)?